### ISSUE TITLE
Enabling JS minification breaks the checkout

### ISSUE THREAD

**Issue by Taintedmedialtd** [Worldpay/worldpay-magento2#11]
Enabling JS minification stops the checkout working by changing the request URL to the WorldPay CDN. Normally, the file requested by requirejs is https://cdn.worldpay.com/v1/worldpay.js but if you enable minification it requests https://cdn.worldpay.com/v1/worldpay.min.js which responds with a 403 error.

**Comment by razvan-f** [comment]
Having more or less the same problem, when JS minification is enabled the Worldpay payment options disappear from the checkout.

**Comment by djam90** [comment]
Yes we also have this issue. Someone else raised the same issue in January. Why aren't Worldpay maintaining this module?
